ci-operator/step-registry/storage/conf/csi-optional/smb-winserver/storage-conf-csi-optional-smb-winserver-commands.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export STORAGECLASS_LOCATION=${SHARED_DIR}/smb-winserver-sc.yaml
```

Comment:
Looks as leftover. Can we remove this line?

Reply:
Good catch! I forgot to remove it after change logic.